zipkin-storage/cassandra-v1/src/main/java/zipkin2/storage/cassandra/v1/CassandraSpanStore.java

```diff
@@ -67,8 +64,6 @@
     indexFetchMultiplier = storage.indexFetchMultiplier;
     strictTraceId = storage.strictTraceId;
     searchEnabled = storage.searchEnabled;
-    timestampCodec = new TimestampCodec(metadata.protocolVersion);
-    buckets = IntStream.range(0, storage.bucketCount).boxed().collect(Collectors.toSet());
```

Comment:
both of these complicated sharing and testing for things that were constant in practice. The timestamp codec has not difference based on version, and bucketCount was always set to 10 with no means to override it ever. since this is a legacy codebase I think being more obvious about constants is better than looking as if we had really parameterized them.